docker/Dockerfile.pdo-build

```diff
@@ -127,12 +127,12 @@ RUN \
 # - build script install in virtualenv. For docker this seems a bit overkill but easier than to
 #   duplicate all build instructions.
  && cd /project/pdo/src/private-data-objects \
- && sawtooth/bin/build_sawtooth_proto \
+# && sawtooth/bin/build_sawtooth_proto \
  && cd python \
```

Comment:
why would you leave these in the file? we're getting rid of sawtooth. remove all vestiges!

Reply:
This is in an earlier commit, but eventually removed in the PR.

Reply:
clean up the commit history before final merge, please.

Reply:
I'd be ok with a squash-and-merge. Would that work?

Reply:
squashing the whole set of commits into one? no. make the commit history relevant. one commit per significant change. every commit is "meaningful".

Reply:
thought we were removing these in this commit?